src/main/core.js

```diff
@@ -122,7 +121,6 @@ function coreFormat(text, opts, addAlignmentSize) {
       newCursorNodeStart = result.cursorNodeStart;
       newCursorNodeText = result.cursorNodeText;
     } else {
-      oldCursorNodeStart = 0;
```

Comment:
Can you explain why this isn’t needed anymore?

Reply:
we have reassigned the variable two times...
 `oldCursorNodeStart = 0; ` and `oldCursorNodeText = text;` that means the first assignment doesnt make sense...

Reply:
I see, it’s never used outside of the `if` branch. Can you move the `let` to the assignment in the `if` branch?

Reply:
@j-f1 if I understood right what u mentioned, oldCursorNodeStart is used in "if", "else" and outside as well... hence it can be declared outside like the way it is...

Reply:
Where is it used outside of the `if` block? I can’t seem to find those uses.